anymail_history/utils.py

```diff
@@ -1,38 +1,81 @@
 from __future__ import annotations
 
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
 from anymail.exceptions import AnymailAPIError
 from anymail.message import AnymailMessage
 from anymail.utils import get_anymail_setting
+from django.core.mail import EmailMultiAlternatives
+from django.template.exceptions import TemplateDoesNotExist
 from django.template.loader import render_to_string
+from html2text import html2text
+
+if TYPE_CHECKING:
+
+    class MultiAlternativesMixin(EmailMultiAlternatives):
+        ...
+
+else:
+
+    class MultiAlternativesMixin:
+        ...
+
+
+class HtmlMessageMixin(MultiAlternativesMixin):
+    def __init__(
+        self,
+        template_name: str,
+        context: dict[str, Any] | None = None,
+        *args: Any,
+        **kwargs: Any,
+    ):
```

Comment:
Is it worth defining a Protocol to explain this Mixin is meant to be used on an instance of EmailMultiAlternatives...